libs/platform/ebpf_epoch.c

```diff
@@ -133,6 +136,8 @@ ebpf_epoch_initiate()
     _ebpf_epoch_cpu_count = cpu_count;
     _ebpf_flush_timer_set = 0;
 
+    C_ASSERT(sizeof(ebpf_epoch_cpu_entry_t) % EBPF_CACHE_LINE_SIZE == 0);
```

Comment:
I believe this macro doesn't need to be inside a function.  Move up right after line 57 so the reason for the padding field is clear there.